src/trace-server.ts

```diff
@@ -199,11 +199,17 @@ export class TraceServer {
     }
 
     private setStatusIfAvailable(started: boolean) {
-        const fromTraceExtension = 'serverStatus';
-        if (started) {
-            vscode.commands.executeCommand(fromTraceExtension + '.started');
-        } else {
-            vscode.commands.executeCommand(fromTraceExtension + '.stopped');
-        }
+        const commands = vscode.commands.getCommands();
+        commands.then((rest) => {
+            const fromTraceExtension = 'serverStatus';
+            // if (rest.findIndex(val => val === (fromTraceExtension + '.started')) > 0) {
```

Comment:
ups, that check is actually needed to avoid the error. The error happens if this extension is installed and the vscode-trace-extension is NOT installed. I'll put it back. Thanks.